### ISSUE TITLE
LDM Training

### ISSUE THREAD

**Issue by Kageyoshi317** [huggingface/diffusers#2164]
Hello,
I tried to adapt your example 
"https://colab.research.google.com/github/huggingface/notebooks/blob/main/diffusers/training_example.ipynb#scrollTo=83158bea-273e-4088-a75a-e3c7cc86e0fc"
which generates buterflies to make it work in latent space with an 
AutoencoderKL.from_pretrained("CompVis/stable-diffusion-v1-4", subfolder="vae")
I changed the initial configuration so that images are transformed to (3, 128,128) which the VAE turns to (4, 16, 16).
The rest is pretty much similar but the resulting images are nothing like butterflies, they are purely orange/yellow with a but of blur here and there.
I would have expected 100 epochs to be enough but I don't know if this is actually the issue. I'd like to avoid training again for 1k epochs for nothing if issue is not epoch related.
The denoiser used consists of Encoder = {down,down,down,attnDown,down} and the Decoder = {up,attnup,up,up,up}.
Have you got any idea? Thanks in advance
![image](https://user-images.githubusercontent.com/122362383/215499170-044c5657-e736-43e0-8d06-ce50654e72bd.png)

**Comment by patrickvonplaten** [comment]
Not really sure, but seems like the loss is nicely going down. Maybe your model is overfitting?

Otherwise maybe the community can help here :-)

**Comment by Kageyoshi317** [comment]
[Fixed]
It was just a matter of latents normalization and also, having a low stable loss seems to be a necessary but not sufficient condition to generate target distribution samples. I kept running for a few more epochs even though the loss was steady and after a few it eventually led to realistic samples.
If someone has an explanation or a redirection to a useful paper, i'm down for it!

**Comment by drimeF0** [comment]
hello, can i ask you for a sample code and/or a learning log?

**Comment by Fouksi** [comment]
> [Fixed] It was just a matter of latents normalization and also, having a low stable loss seems to be a necessary but not sufficient condition to generate target distribution samples. I kept running for a few more epochs even though the loss was steady and after a few it eventually led to realistic samples. If someone has an explanation or a redirection to a useful paper, i'm down for it!

What change did you make in the latent normalization?
